commands/cli/cmd_suggestion.go

```diff
@@ -0,0 +1,71 @@
+package cli
+
+import (
+	"sort"
+	"strings"
+
+	cmds "github.com/ipfs/go-ipfs/commands"
+	levenshtein "github.com/texttheater/golang-levenshtein/levenshtein"
```

Comment:
will want to vendor. run 
```
make vendor
```
from root.

---

in the event that godeps barfs at you with all sorts of requests, try `go get` on all the needed packages. 

people report that this works too:
```
godep restore
```
or even setting
```
export GOPATH=`pwd`/Godeps/_workspace:$GOPATH
```